### ISSUE TITLE
Swipe stops when I get just try to divide the tab bar

### ISSUE THREAD

**Issue by Ruzen1** [GeekyAnts/NativeBase#3764]
Hey Native Base team,
 Thank you for the library.

We are trying to make a tab bar with scrolling and swiping feature. The provided Scrollable Tab is awesome and works fine. 
However, we wanted to keep another option which is also a tab button but not used from the native base  just beside the tab bar. When we keep it, the swipe feature doesnt function anymore.
Can somebody please give an idea?  Thank you

**Comment by tewarig** [comment]
hi, maybe can you share a mock-up of what you are trying to create. or just share the code in the expo snack where the swipe didn't work. maybe I could help you with that

**Comment by Ajamuar** [comment]
@Ruzen1 Is this for version 3?

**Comment by Ruzen1** [comment]
@tewarig @Aj
 
 Thank you very much for your comments. 
 I actually solved it by some other alternative way. The desired target was to keep the ellipsis fixed as you see in the below picture at the rightmost of the tab and make other tabs scrollable excluding the ellipisis.
 
 However I made it possible by keeping the ellipsis just above the tab bar and not as a part of tab bar and made the gave some marginRight to the last item of the tabs .

So I will close the issue for now. Would be glad if there was such kind of feature in this library.

<img width="362" alt="スクリーンショット 2021-07-01 11 59 34" src="https://user-images.githubusercontent.com/25242058/124058167-ce23c400-da63-11eb-86c9-06038ff504a4.png">
